### PR TITLE
Upgrades to RN 54

### LINKED ISSUE #38: Update to RN 54

### PR DESCRIPTION
**Pull request by lXSPandora**
closes #38

### CODE REVIEW

File: src/components/Input.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const RegisterTextInput = styled.TextInput`
   height: 40;
   width: 100%;
   font-size: 20;
+  padding-top:2;
```
Comment (lucasbento) [inline src/components/Input.js]: Missing space before `:`.

File: scripts/redirect_ports.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# packager
+adb reverse tcp:8081 tcp:8081
+adb -d reverse tcp:8081 tcp:8081
+adb -e reverse tcp:8081 tcp:8081
+
+# Storybooks
```
Comment (lucasbento) [inline scripts/redirect_ports.sh]: This comment should be in singular, shouldn't it?

File: package.json [hunk 3]
```diff
@@ -10,8 +10,8 @@
   "dependencies": {
     "babel-preset-react-native-stage-0": "^1.0.1",
     "hoist-non-react-statics": "^2.5.0",
-    "react": "16.0.0-alpha.6",
-    "react-native": "0.43.4",
+    "react": "16.3.0-alpha.1",
```
Comment (lucasbento) [inline package.json]: hmm

File: src/components/Input.js (hunk 1 above)
Comment (lXSPandora) [reply src/components/Input.js]: nice i will do it

**Pull request merged**